### ISSUE TITLE
ch3 : {{{body}}} for handlebar

### ISSUE THREAD

**Issue by alireza-saberi** [EthanRBrown/web-development-with-node-and-express#42]
I came from front-end to back-ebd `{{{body}}}`of handlebar is close to data-binding in AngularJS and it causes troubles. Any sugesstion/replacement except Jade?

**Comment by EthanRBrown** [comment]
This won't conflict with Angular as long as the rendering is done on the back-end.  If you're doing front-end rendering with Handlebars, that could cause a conflict, but if you're using Angular, chances are you don't need or want Handlebars on the FE.

So if you use `{{{body}}}` on the server side, that will be replaced before it's sent over the wire to the browser, where Angular is working its magic.  If, on the other hand, you need Handlebars _not_ to replace double or triple curly brackets (for example, if you want them passed to the client for use in Angular data binding), you can simply escape them with a backslash.  Consider the following back-end Handlebars template with the context object `{ nonny: '<b>nonny</b>' }`:

```
Back end:                 Rendered to client:
-----------------         -------------------------------
hey {{nonny}} no          hey &lt;b&gt;nonny&lt;b&gt; no
hey \{{nonny}} no         hey {{nonny}} no
hey {{{nonny}}} no        hey <b>nonny</b> no
hey \{{{nonny}} no        hey {{{nonny}}} no
```

In other words, if you render your Handlebars on the back end, no matter what you want to do, you can do it!  While it's probably not impossible to combine Handlebars and Angular on the FE, that would be a bigger headache and should probably be avoided.
